raiden/tests/smart_contracts/test_netting_channel.py

```diff
@@ -1018,3 +1018,123 @@ def test_settlement_with_unauthorized_token_transfer(
     assert tester_token.balanceOf(address1, sender=privatekey1_raw) == initial_balance1 + deposit + transfer_amount0 - transfer_amount1  # noqa
     # Make sure that the extra amount is burned/locked in the netting channel
     assert tester_token.balanceOf(nettingchannel.address, sender=privatekey1_raw) == extra_amount
+
+
```

Comment:
Since you are using tester functionality make this tests runnable only by tester by adding:
```
@pytest.mark.parametrize('blockchain_type', ['tester'])
```